ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -18,21 +19,24 @@ class PurchasesHybridCommonTests: QuickSpec {
         context("proxy url string") {
             it("parses the string and sets the url if valid") {
                 let urlString = "https://revenuecat.com"
-                RCCommonFunctionality.proxyURLString = urlString
+                CommonFunctionality.proxyURLString = urlString
                 expect(Purchases.proxyURL?.absoluteString) == urlString
             }
 
             it("sets the proxy to nil if null passed in") {
                 let urlString = "https://revenuecat.com"
-                RCCommonFunctionality.proxyURLString = urlString
+                CommonFunctionality.proxyURLString = urlString
                 expect(Purchases.proxyURL).toNot(beNil())
 
-                RCCommonFunctionality.proxyURLString = nil
+                CommonFunctionality.proxyURLString = nil
                 expect(Purchases.proxyURL).to(beNil())
             }
 
             it("raises exception if proxy can't be parsed") {
-                expect { RCCommonFunctionality.proxyURLString = "not a valid url" }.to(raiseException())
+                let expectedMessage = "could not set the proxyURL, provided value is not a valid URL: not a valid url"
+                self.expectFatalError(expectedMessage: expectedMessage) {
+                    CommonFunctionality.proxyURLString = "not a valid url"
+                }
```

Comment:
can't check for `fatalError` with Nimble on apple silicon, so I copied our workaround from purchases-ios